lib/quicly.c

```diff
@@ -2760,18 +2760,17 @@ static int handle_ack_frame(quicly_conn_t *conn, size_t epoch, quicly_ack_frame_
     }
     if (cc_type != 0)
         cc_cong_signal(&conn->egress.cc.ccv, cc_type, bytes_in_flight);
-    int end_of_recovery = frame->largest_acknowledged >= conn->egress.cc.end_of_recovery;
+    int exit_recovery = frame->largest_acknowledged >= conn->egress.cc.end_of_recovery;
```

Comment:
Is end-of-recovery also being populated somewhere?

Reply:
`end_of_recovery` is the PN that marks the end of recovery. `exit_recovery` is a boolean indicating if the congestion controller is exiting recovery.

The two values are logged under the same names. `end-of-recovery` is logged when congestion is detected. `exit-recovery` is logged when a new ACK is received.